### ISSUE TITLE
Write a Dev-focused article about How to build with ChiConnect, Chimoney's API

### ISSUE THREAD

**Issue by uchibeke** [Chimoney/chimoney-community-projects#17]
This is a good first issue that anyone can pick up

Feel free to come up with any blog idea like How to Build Mobile Apps, Sending Payments with Chimoney etc

Please share your idea in the comment below if you'd like our thoughts before going ahead to write it

### Resources
- About ChiConnect [chimoney.io/chiconnect](chimoney.io/chiconnect)
- Reference [chimoney.readme.io/reference/introduction](chimoney.readme.io/reference/introduction)
- Swagger Docs [api.chimoney.io/api-docs](api.chimoney.io/api-docs)

**Comment by Hauwarh** [comment]
Hi,
I will love to work on this issue, please assign this to me.
Thanks

**Comment by uchibeke** [comment]
Please create a new issue for the specific article you want to write

Or share your article details here and I will create the issue and assign to you

**Comment by Khadeeejah** [comment]
hi @uchibeke  can more than one person work on this ?

**Comment by uchibeke** [comment]
@Khadeeejah many people can work on this. They can create a new issue with details of the content they want to contribute

**Comment by Adelakin-Adewumi** [comment]
@uchibeke I'd love to also contribute to this article, is that okay?

**Comment by uchibeke** [comment]
> @uchibeke I'd love to also contribute to this article, is that okay?

Sure. Please create a new issue with details of the content you want to contribute @Adelakin-Adewumi

**Comment by Conradgabe** [comment]
Hi, @uchibeke I'd love to work on this task

**Comment by uchibeke** [comment]
> Hi, @uchibeke I'd love to work on this task

Hi please[ create a new issue](https://github.com/Chimoney/chimoney-community-projects/issues/new) with the details of the article you want to write @Conradgabe and assign it to yourself

**Comment by uchibeke** [comment]
Hi @Hauwarh, @Khadeeejah @Adelakin-Adewumi could you[ create a new issue](https://github.com/Chimoney/chimoney-community-projects/issues/new) with the details of the article you want to write and ask any questions by commenting there

**Comment by Conradgabe** [comment]
Good day, @uchibeke, the issue says "How to build", do you mean the process of building an application with the API functionalities?

**Comment by uchibeke** [comment]
Yes @Conradgabe. Could you crate the issue and mention me? I’ll assign to you?

**Comment by larryease** [comment]
@uchibeke, all the links associated with resources have a 404 error. How do we get the documentation for the blog?

**Comment by uchibeke** [comment]
@larryease please try again

**Comment by larryease** [comment]
Accessible now. Thanks @uchibeke

**Comment by SHAIKASIFALI** [comment]
Hi can I contribute to this in hacktoberfest 2023
